spec/facility_spec.rb

```diff
@@ -23,4 +31,35 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
+
+      # binding.pry
+
+      expect (@facility_1.register_vehicle(@cruz)).should == nil
+      @facility_1.add_service( 'Vehicle Registration' )
+      expect (@facility_1.registered_vehicles).should == []
+      expect (@cruz.plate_type).should == nil
+      expect (@cruz.registration_date).should == nil
+      expect (@facility_1.collected_fees).should == 0
+      expect (@facility_1.register_vehicle(@cruz)).should == [@cruz]
+      # expect (@cruz.plate_type).should == :regular
+      expect (@cruz.registration_date).should == Date.today
+      expect (@facility_1.collected_fees).should == 100
+      # expect(@facility_1.registered_vehicles).should == [@cruz]
+    end
+  end
+
+  describe '#renew_drivers_license' do
```

Comment:
Same case here -- using the interaction pattern, try to build out a few test cases to cover the different scenarios:
- registrant cannot renew if the facility doesn't have that service
- registrant cannot renew if they don't have written and license exam set to true
- registrant CAN renew if written and license is true and facility has exam service

These tests will be shorter than what you have here because you're trying to test all the scenarios in one. For simple methods (i.e one simple if statement) that's okay, but this is a more complex method so breaking it up into multiple `it` blocks would be better for troubleshooting, readability and robustness.